solutions/USACO dec20-cowntagion.mdx

```diff
@@ -2,14 +2,69 @@
 id: usaco-1062
 source: USACO Silver 2020 December
 title: Cowntagion
-author: Kevin Sheng
+author: Kevin Sheng (Java, Python), Tanish Tyagi (C++)
 ---
 
 [Official Editorial](http://www.usaco.org/current/data/sol_prob1_silver_dec20.html)
 
 <LanguageSection>
 
+<CPPSection>
+
+## C++ Implementation
+
+```cpp 
+#include <bits/stdc++.h>
+
+using namespace std;
+
+#define maxn 100005
+int n;
+vector<int> adj[maxn];
+
+int dfs(int start, int parent) {
+	int ans = 0;
+	int cows = adj[start].size();
+	if (parent == -1) cows++;
+
+	int currCows = 1;
+	int days = 0;
+	//count the number of superspreader events needed
+	while (currCows < cows) {
+		days++;
+		currCows *= 2;
+	}
+	ans += days;
+
+	for (auto next : adj[start]) {
+		if (next != parent) {
+			//add 1 because the sick cow takes 1 day to travel to next node
```

Comment:
Once you address my request (above), delete this comment, since it'll be redundant.